src/main/java/io/openliberty/tools/common/plugins/config/ServerConfigDocument.java

```diff
@@ -180,6 +180,12 @@ public static ServerConfigDocument getInstance(CommonLoggerI log, File serverXML
         return instance;
     }
 
+    public static void setLogger(CommonLoggerI log) {
+        ServerConfigDocument.log = log;
```

Comment:
The `log` is now non-static as are the other methods. Need to add a new constructor that takes the minimum parms needed, and instantiate this class to call the other methods.